.buildkite/test-pipeline.yaml

```diff
@@ -39,6 +39,7 @@ steps:
 
 - label: Models Test
   commands:
+    - bash ../.buildkite/download-images.sh
     - pytest -v -s models --forked
   soft_fail: true
```

Comment:
oof this is a bit tough because it is soft failed. do you think the test can run on a single L4 (with fp16)? If so maybe we can create another job for the test that are currently passing right now.

Reply:
@simon-mo is Llava support coming?

Reply:
could you say more about this point? I think the current CI should pass. It's still failing on the same huggingface error.  I am confused about that.